redisinsight/ui/src/pages/rdi/pipeline/components/tab/Tab.tsx

```diff
@@ -35,6 +37,7 @@ const Tab = (props: IProps) => {
         >
           <EuiIcon type="document" className="rdi-pipeline-nav__fileIcon" />
           <EuiText className="rdi-pipeline-nav__text">{fileName}</EuiText>
+          {isLoading && <EuiLoadingSpinner data-testid="rdi-nav-config-loader" className={styles.loader} />}
```

Comment:
do you think it's necessary to have a unit test that checks for the loading indicator when `isLoading = true`?

Reply:
I don't think this test is necessary. But i can if needed